CHANGELOG.rst

```diff
@@ -3,42 +3,46 @@
 
 **Incompatible changes**
 
-* Removed the following deprecated commandline options
+A number of incompatible changes were made in this release, with the intent of removing features deprecated for a long
+time or change existing behaviors in order to make them less surprising/more useful.
 
-  * ``--genscript``
-  * ``--no-assert``
-  * ``--nomagic``
-  * ``--report``
+* The following deprecated commandline options were removed:
 
-  Thanks to `@RedBeardCode`_ for the PR (`#1664`_)
+  * ``--genscript``: no longer supported;
+  * ``--no-assert``: use ``--assert=plain`` instead;
+  * ``--nomagic``: use ``--assert=plain`` instead;
+  * ``--report``: use ``-r`` instead;
+
+  Thanks to `@RedBeardCode`_ for the PR (`#1664`_).
 
 * ImportErrors in plugins now are a fatal error instead of issuing a
   pytest warning (`#1479`_). Thanks to `@The-Compiler`_ for the PR.
 
-* removed support code for python 3 < 3.3 addressing (`#1627`_)
+* Removed support code for Python versions < 3.3 addressing (`#1627`_).
```

Comment:
Also the parens in "addressing (`#1234`)" don't make any sense. I'm really not sure where this came from - maybe because older changelog entries said "[Added a] fix (...) for some issue" and then it got [cargo-culted](https://en.wikipedia.org/wiki/Cargo_cult_software_engineering) and also used like "Fixes (...)" and "addressing (...)". Anyways, we should get rid of it IMHO.